run_dir/design/ont_flowcells.html

```diff
@@ -47,13 +55,40 @@ <h1 class="ont_fc_page_heading" id="ont_page_title">ONT Flowcells</h1>
                 <abbr title="Ongoing"><span class="fa fa-hourglass-half"></span></abbr>
             {% end %}
         </td>
-        <td>{{ ont_flowcells[onefc]["start_date"] }}</td>
+
+
+        {% if find_id(ont_flowcells[onefc]['sample_name'], 'sample') %}
+            <td class="sample">
+                <a class="text-decoration-none" href="https://ngi-lims-prod.scilifelab.se/clarity/search?scope=Sample&query={{ find_id(ont_flowcells[onefc]['sample_name'], 'sample') }}">
```

Comment:
Bonus points if this link points to lims-stage if genstat is running from statusdb dev...

Reply:
[93b885a](https://github.com/SciLifeLab/genomics-status/pull/740/commits/93b885adb1c6aa6b7eccb8759d5b8da9a4745a6d)